rclcpp/CMakeLists.txt

```diff
@@ -144,12 +144,15 @@ if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
 
+  find_package(rcpputils REQUIRED)
```

Comment:
`rcpputils` is already find_package'd above, so I don't think this is necessary.